src/core/flow.js

```diff
@@ -9,6 +9,36 @@
 import { TurnOrder } from './turn-order';
 import { Random } from './random';
 
+/**
+ * This function checks whether a player is allowed to make a move.
+ *
+ * @param {object} G     - The Game instance
+ * @param {object} ctx   - The ctx instance
+ * @param {object} opts  - Options - used here to transport the playerID.
+ */
+function canMakeMoveDefault(G, ctx, opts) {
+  const { playerID } = opts || {};
+
+  // In multiplayer mode, the default playerID is null, which corresponds
+  // to a spectator that can't make moves.
+  if (playerID === null) {
+    return true;
```

Comment:
Hello! I was reading through the codebase and stubbled across this line. The `true` value here seems in contradiction with the comment right before. If the spectator can _not_ make moves, shouldn't this return `false` instead?

Reply:
This is now fixed. Thanks for the report.